invenio_app_rdm/theme/views.py

```diff
@@ -94,3 +96,26 @@ def help_statistics():
             "invenio_app_rdm/help/statistics.en.html",
         ]
     )
+
+
+def notification_settings():
+    """View for notification settings."""
+    preferences_notifications_form = NotificationsForm(
+        formdata=None, obj=current_user, prefix="preferences-notifications"
+    )
+
+    # Pick form
+    form_name = request.form.get("submit", None)
+    form = preferences_notifications_form if form_name else None
```

Comment:
comment: I guess this is for future-proofing this function for when we add more sections/forms to the page